### ISSUE TITLE
Can not visit ‘Access control’ page

### ISSUE THREAD

**Issue by flyher** [shadowsocks/luci-app-shadowsocks#134]
1.63 worked well ,but the 1.7,and 1.8 version show the log when i open 'Access control' page.
openwrt 14

```log
/usr/lib/lua/luci/dispatcher.lua:448: Failed to execute cbi dispatcher target for entry '/admin/services/shadowsocks/access-control'.
The called action terminated with an exception:
...ib/lua/luci/model/cbi/shadowsocks/access-control.lua:92: attempt to call field 'neighbors' (a nil value)
stack traceback:
	[C]: in function 'assert'
	/usr/lib/lua/luci/dispatcher.lua:448: in function 'dispatch'
	/usr/lib/lua/luci/dispatcher.lua:195: in function </usr/lib/lua/luci/dispatcher.lua:194>
```
related issue:
#126

**Comment by aa65535** [comment]
Upgrade your firmware is a better choice.

**Comment by EvianZhow** [comment]
@aa65535 Same issue here. So old firmware better stay at 1.6.3 version?

**Comment by aa65535** [comment]
@EvianZhow already fixed in c13e285

**Comment by flyher** [comment]
@aa65535 
Thank you very much!
